### DISCUSSION TITLE
How to plot on top of existing figures?

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#733] [Q&A] [answered]
Hi,

Could you show me how to plot on top of existing figures?
I find that you guys have a standalone mode.
But I cannot use it.
Here is my code
```
colour.plotting.plot_chromaticity_diagram_CIE1931(standalone=False)
plt.scatter(xy[:100,0], xy[:100,1])
colour.plotting.render(
    standalone=True,
    limits=(-0.1, 0.9, -0.1, 0.9),
    x_tighten=True,
    y_tighten=True)
```
![image](https://user-images.githubusercontent.com/16201681/73629328-bfdc3900-4620-11ea-84ae-d4bafcbeb7d9.png)
![image](https://user-images.githubusercontent.com/16201681/73629338-c79bdd80-4620-11ea-83f5-189bc14661d8.png)
![image](https://user-images.githubusercontent.com/16201681/73629349-ce2a5500-4620-11ea-9f92-b093a8dc17fc.png)

What should I do in my case?
Thanks,

**Comment by thomasmansencal** [comment, marked as answer]
Hi @hminle,

Jupyter Notebook tends to display the figure/axes immediately if the last result from a call is a figure or an axes. Try to put everything in a single cell.

**Comment by hminle** [comment]
Hi @KelSolaar ,

It works. Thank you a lot :)
![image](https://user-images.githubusercontent.com/16201681/73662457-d22e9500-4669-11ea-8ee5-75544cce6714.png)

**Comment by thomasmansencal** [comment]
Note that if you are interested with plotting image chromaticities there is that definition that does the heavy lifting for you: https://colour.readthedocs.io/en/develop/generated/colour.plotting.plot_RGB_chromaticities_in_chromaticity_diagram_CIE1931.html#colour.plotting.plot_RGB_chromaticities_in_chromaticity_diagram_CIE1931

https://colab.research.google.com/drive/1NRcdXSCshivkwoU2nieCvC3y14fx1X4X#scrollTo=6PlESt8vGeeT

**Comment by hminle** [comment]
> Note that if you are interested with plotting image chromaticities there is that definition that does the heavy lifting for you: https://colour.readthedocs.io/en/develop/generated/colour.plotting.plot_RGB_chromaticities_in_chromaticity_diagram_CIE1931.html#colour.plotting.plot_RGB_chromaticities_in_chromaticity_diagram_CIE1931
> 
> https://colab.research.google.com/drive/1NRcdXSCshivkwoU2nieCvC3y14fx1X4X#scrollTo=6PlESt8vGeeT

@KelSolaar Thank you very much. It's exactly what I need 👍 👍

**Comment by hminle** [comment]
@KelSolaar you guys should put the colab notebook here
![image](https://user-images.githubusercontent.com/16201681/73746570-76c0dd80-4723-11ea-9cbe-4bade6980189.png)
It's super helpful 👍

**Comment by thomasmansencal** [comment]
So it should be linked here close to the top!

 https://colour.readthedocs.io/en/develop/#how-to-guide

**Comment by hminle** [comment]
> So it should be linked here close to the top!
> 
> https://colour.readthedocs.io/en/develop/#how-to-guide

oh nice, I did not notice about it. Thank again :)
